MoreLinq.Test/SkipUntilTest.cs

```diff
@@ -57,5 +59,25 @@ public void SkipUntilEvaluatesPredicateLazily()
             var sequence = Enumerable.Range(-2, 5).SkipUntil(x => 1 / x == -1);
             sequence.AssertSequenceEqual(0, 1, 2);
         }
+
+        public static readonly IEnumerable<ITestCaseData> TestData =
+            from e in new[]
+            {
+                new {s = new int[0]     , o = 0, r = new int[0]},   // empty sequence
```

Comment:
Can we use more meaningful (Pascal-case) names for the properties?

```diff
-                new {s = new int[0]     , o = 0, r = new int[0]},   // empty sequence
+                new { Source = new int[0]     , Min = 0, Expected = new int[0] },   // empty sequence
```

Also, space after `{` and before `}`.